cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractConfiguration.java

```diff
@@ -77,8 +77,11 @@ public AbstractConfiguration(String configuration, List<String> forbiddenOptions
 
             if (value instanceof String)    {
                 map.put(key, (String) value);
-            } else if (value instanceof Integer || value instanceof Long || value instanceof Boolean || value instanceof Double || value instanceof Float)    {
+            } else if (value instanceof Integer || value instanceof Long || value instanceof Boolean || value instanceof Double || value instanceof Float) {
                 map.put(key, String.valueOf(value));
+            } else if (value == null) {
+                log.error("A null value for key {} is not allowed", key);
+                throw new InvalidConfigMapException(key, "A null value is not allowed for this key");
```

Comment:
Why is this `InvalidConfigMapException`? I though that is used for logging the external config map is invalid. If it is used more widely? should we rename it?

Reply:
Well, I think that it's not so related to ConfigMap which now is Custom Resource right? Because it's related to a "configuration" what do you think about `InvalidConfigParameterException`?

Reply:
Yeah ... that would sound better and more easier to understand. Or use the `InvalidConfigurationException` which is in `operator-common`

Reply:
I think it's better than reusing the already existing one.

Reply:
And I'm going to remove `InvalidConfigMapException` which will not be used anymore.